doc/news/_preparation_next_release.md

```diff
@@ -331,384 +249,181 @@ This section keeps you up-to-date with the multi-language support provided by El
-- <<TODO>>
-- <<TODO>>
+- Decide and implement [decision process](../decisions/5_partially_implemented/decision_process.md). _(Markus Raab)_
+- Decided future [library split](../decisions/4_decided/library_split.md). _(@kodebach)_
+- Decided [decision process](https://www.libelektra.org/decisions/decision-process). _(Markus Raab)_
+- Draft for [man pages](../decisions/0_drafts/man_pages.md). _(Markus Raab)_
+- Add decision for [change tracking](../decisions/1_problem_clear/change_tracking.md). _(Maximilian Irlinger @atmaxinger)_
+- Create [decision](../decisions/0_drafts/operation_sequences.md) for allowed and prohibited operation seqences. _(Maximilian Irlinger @atmaxinger)_
+- Add decisions about [location of headers](../decisions/4_decided/header_file_structure.md) and [use of `#include`](../decisions/4_decided/header_include.md) in the repo. _(@kodebach)_
+- Add decision about [metadata semantics](../decisions/0_drafts/metakey_semantics.md). _(@kodebach)_
 - Many small fixes to adapt to documentation guidelines and new decision process. _(Markus Raab)_
-- <<TODO>>
-- Add decision for [read-only keynames](../decisions/0_drafts/readonly_keynames.md) _(Maximilian Irlinger @atmaxinger)_
-- <<TODO>>
-- <<TODO>>
-- Revive [keyname decision](../decisions/4_decided/keyname.md) _(@kodebach)_
-- <<TODO>>
-- <<TODO>>
+- Add decision for [read-only keynames](../decisions/0_drafts/readonly_keynames.md). _(Maximilian Irlinger @atmaxinger)_
+- Revive [keyname decision](../decisions/4_decided/keyname.md). _(@kodebach)_
 - Add decision for [copy-on-write](../decisions/6_implemented/copy_on_write.md) and provide implementation suggestions. _(Maximilian Irlinger @atmaxinger)_
-- <<TODO>>
-- <<TODO>>
-- <<TODO>>
 - Add decisions for [constructor functions](../decisions/0_drafts/constructor_functions.md) and [builder functions](../decisions/0_drafts/builder_functions.md). _(@kodebach)_
-- <<TODO>>
-- <<TODO>>
-- <<TODO>>
-- Update [internal cache](../decisions/4_decided/internal_cache.md) _(Markus Raab)_
-- Move [internal cache](../decisions/4_decided/internal_cache.md) back to draft _(@kodebach)_
-- <<TODO>>
-- <<TODO>>
-- Create [transformations](../decisions/0_drafts/transformations.md) _(Maximilian Irlinger @atmaxinger)_
-- <<TODO>>
-- <<TODO>>
-- Replace TOC-style [README.md](../decisions/README.md) with folders and generate HTML for website _(@kodebach)_
-- <<TODO>>
-- <<TODO>>
-- <<TODO>>
-- Restructured decisions directories based on new agreed-upon [steps](../decisions/STEPS.md) _(Maximilian Irlinger @atmaxinger)_
-- <<TODO>>
-- <<TODO>>
-- Decision for [types of `KeySet`s](../decisions/2_solutions_clear/keyset_types.md) _(@kodebach)_
-- <<TODO>>
-- Added [Documentation Guidelines](https://master.libelektra.org/doc/contrib/documentation.md) _(Markus Raab)_
+- Update [internal cache](../decisions/4_decided/internal_cache.md). _(Markus Raab)_
+- Move [internal cache](../decisions/4_decided/internal_cache.md) back to draft. _(@kodebach)_
```

Comment:
Are those two not redundant? It's now in `decided`.

Reply:
You're right, looks weird for the final release notes.

Both entries were probably true at some point in time.